src/BuildScriptGenerator/PlatformInstallerBase.cs

```diff
@@ -195,17 +198,6 @@ public virtual void InstallPlatformSpecificSkeletonDependencies(StringBuilder st
                 .AppendLine($"  fi")
                 .AppendLine("fi")
 
-                // Required for python 3.10 and 3.11
-                .AppendLine($"PIP_GCC_FLAGS=\"\"")
-                .AppendLine($"if [ \"$platformName\" = \"python\" ]; then")
-                .AppendLine($"  if [[ '{version}' == 3.10* ]] || [[ '{version}' == 3.11* ]]; then")
-                .AppendLine($"    echo \"Enabling --global-option to pip for gcc modules...\"")
-                .AppendLine($"    pythonMajorMinorVersion={version.Substring(0, version.LastIndexOf('.'))}")
-                .AppendLine($"    dynamicInstallationPythonIncludeDirectory={versionDirInTemp}/include/python$pythonMajorMinorVersion")
-                .AppendLine($"    PIP_GCC_FLAGS=\"--global-option=build_ext --global-option=-I$dynamicInstallationPythonIncludeDirectory\"")
-                .AppendLine($"  fi")
-                .AppendLine("fi")
-
```

Comment:
Can we keep this for cli and add a condition to skip this if the image is github-actions?

Reply:
@snehapar9 Why do we want this for the CLI image? This entire section is no longer needed since we added the environment variables to `benv.sh`, right?

Reply:
Okay. That makes sense.